k8s.gcr.io/images/k8s-staging-build-image/images.yaml

```diff
@@ -1,6 +1,16 @@
 - name: kube-cross
   dmap:
-    "sha256:97e77164ddc1f9ab56bcbab3e2895b882097b1eabd820497e894f41c0e4908fe": ["v1.12.12-1"]
-    "sha256:8d3fd0e3e126b085d4475f1954cc394a2ebea908f370095eb7463d2b1f4dbfdd": ["v1.12.17-1"]
-    "sha256:df0a50772214025040ea31144a731b5fa76944cbfc9c87db05af98e4aa39aa7f": ["v1.13.6-1"]
-    "sha256:0c821cd481e9bbfc9d380290d1401a856e9c2bc0f8d0673fc1519e9bef408d1d": ["v1.13.8-1"]
+    sha256:0c821cd481e9bbfc9d380290d1401a856e9c2bc0f8d0673fc1519e9bef408d1d:
```

Comment:
personally not a fan of making this more lines just because.
what did sorting by digest accomplish ...?

Reply:
guessing this is what a yaml normalizer outputs? (sorting keys, unquoting strings that don't need quotes, etc)

Reply:
(though I tend to agree... makes finding the shas for a specific version or range of versions hard)